src/main/java/marsException.java

```diff
@@ -0,0 +1,5 @@
+public class marsException extends RuntimeException{
+    public marsException(String message){
```

Comment:
```suggestion
public class MarsException extends RuntimeException{
    public MarsException(String message){
```
I noticed that the class name marsException does not follow the PascalCase convention for class names. According to the coding standard, class names should be nouns and written in PascalCase. Consider renaming it to MarsException to align with the standard.